starter-code/views/drones/index.ejs

```diff
@@ -0,0 +1,10 @@
+<h2>Mother of Drones</h2>
+  <ul>
+    <% drones.forEach((aDrone) => { %>
+      <h3><%= aDrone.droneName %></h3>
```

Comment:
👎  you should have:
**Use a forEach loop to display tags with each drone's droneName, propellers, and speed.**